addons/block_code/block_code_plugin.gd

```diff
@@ -59,6 +61,17 @@ func _enter_tree():
 		new_profile.save_to_file(dir)
 		EditorInterface.set_current_feature_profile("block_code")
 
+func script_window_requested(script: String):
+	
+	var script_window = ScriptWindow.instantiate()
+	script_window.script_content = script
```

Comment:
@GustavoLR548 I noticed in your screenshot that there is no indentation. I think you can get indentation by replacing the tabs with 4 spaces:
```suggestion
	script_window.script_content = script.replace('\t', '    ')
```

And then add a monospace font to the label:
- Add Label Settings
- Add Font -> SystemFont
- Font Names -> Add Element -> Monospace
![Captura desde 2024-08-01 13-55-34](https://github.com/user-attachments/assets/0165d465-47da-45fd-a34f-1606ae9ec898)

The result will be something like this:
![Captura desde 2024-08-01 14-00-30](https://github.com/user-attachments/assets/db74838a-37bd-489b-8938-24f0d9053f51)